L1Trigger/TrackFindingTracklet/src/TrackletProjectionsMemory.cc

```diff
@@ -34,17 +35,22 @@ void TrackletProjectionsMemory::addProj(Tracklet* tracklet) {
 void TrackletProjectionsMemory::clean() { tracklets_.clear(); }
 
 void TrackletProjectionsMemory::writeTPROJ(bool first) {
+  const string dirTP = settings_.memPath() + "TrackletProjections/";
+  if (not std::filesystem::exists(dirTP)) {
+    system((string("mkdir -p ") + dirTP).c_str());
```

Comment:
check if successful

Reply:
DONE
--  P.S. As this file/directory creation code appears in dozens of places, a future PR (almost ready) will move it to a single common function, to reduce duplicate code.